install/samples/lazyload/slimeboot_cluster_accesslog.yaml

```diff
@@ -0,0 +1,43 @@
+---
+apiVersion: config.netease.com/v1alpha1
+kind: SlimeBoot
+metadata:
+  name: lazyload
+  namespace: mesh-operator
+spec:
+  image:
+    pullPolicy: Always
+    repository: docker.io/slimeio/slime-lazyload
+    tag: master-ecadccb-dirty_8d0e997
+  module:
+    - name: lazyload
+      enable: true
+      fence:
+        wormholePort: # replace to your application service ports, and extend the list in case of multi ports
+          - "80"
+          - "9080"
+        namespace: # replace to your service's namespace which will use lazyload, and extend the list in case of multi namespaces
+          - default
+      global:
+        misc:
+          global-sidecar-mode: cluster # inform the lazyload controller of the global-sidecar mode, default is namespace
+          metric_source_type: accesslog # indicate the metric source
+  component:
+    globalSidecar:
+      enable: true
+      type: cluster # inform the slime-boot operator of the global-sidecar mode
```

Comment:
Is this `type: cluster` duplicated with `global.misc.global-sidecar-mode` in the `lazyload` module configuration?

Reply:
use global.misc.global_sidecar_mode and remove globalSidecar.type now